server/controllers/order.js

```diff
@@ -0,0 +1,94 @@
+const express = require ("express");
+const bodyParser = require("body-parser");
+const ordersModel = require("../model/order");
+
+const app = express();
+
+const Logger = require('../services/logger_services'); 
+const logger = new Logger('app');
+
+app.use(bodyParser.urlencoded({extended: true}));
+app.use(bodyParser.json());
+
+const idGeneretor = {
+    lastId: 0,
+    generateId: function() {
+        return ++(this.lastId);
+    },
+};  
+
+function dateGenerator() {
+	let today = new Date();
+	let dd = String(today.getDate()).padStart(2, "0");
+	let mm = String(today.getMonth() + 1).padStart(2, "0");
+	let yyyy = today.getFullYear();
+
+	today = dd + "/" + mm + "/" + yyyy;
+	return today;
+}
+
+const getAllOrders = (req, res) => { 
+    const ordersList = ordersModel.getAllOrders();
+    logger.setLogData(ordersList);
+    logger.info("Orders were fetched");
```

Comment:
I think it will be better to just use info(message, obj), I need to check, don't change it for now